driver/clirr-ignored-differences.xml

```diff
@@ -433,4 +433,28 @@
         <method>org.neo4j.driver.BookmarkManager queryTaskBookmarkManager()</method>
     </difference>
 
+    <difference>
+        <className>org/neo4j/driver/summary/Notification</className>
+        <differenceType>7012</differenceType>
+        <method>java.util.Optional severityLevel()</method>
+    </difference>
+
+    <difference>
+        <className>org/neo4j/driver/summary/Notification</className>
+        <differenceType>7012</differenceType>
+        <method>java.util.Optional rawSeverityLevel()</method>
+    </difference>
+
+    <difference>
+        <className>org/neo4j/driver/summary/Notification</className>
+        <differenceType>7012</differenceType>
+        <method>java.util.Optional category()</method>
+    </difference>
+
+    <difference>
+        <className>org/neo4j/driver/summary/Notification</className>
+        <differenceType>7012</differenceType>
+        <method>java.util.Optional rawCategory()</method>
+    </difference>
```

Comment:
How does it works?

Reply:
[Clirr is used to raise awareness of binary compatibility.](https://www.mojohaus.org/clirr-maven-plugin/usage.html#Compatibility_Checks)

> The clirr:check goal is used to check for compatibility with previous versions.

In this case it checks against version `5.6.0` and found that 4 new methods have been added to the `Notification` interface, which could break implementations of this interface. In this case, however, it is suggested to accept this change as we don't expect nor support external implementations of the `Notification` interface. We will probably make it `sealed` in upcoming major release.